src/Paths/Directory.ts

```diff
@@ -28,7 +26,7 @@ export const root = "" as Directory;
 const invalidCharacters = ["/", "*", "?", "[", "]", "!", "\\", "~", "|", "#", '"', "'", " "];
 
 /** A valid character is any character that is not one of the invalid characters */
-export const oneValidCharacter = `[^${escapeRegExp(invalidCharacters.join(""))}]`;
+export const oneValidCharacter = `[^${invalidCharacters.map((c) => "\\" + c).join("")}]`;
```

Comment:
Not all of these form valid escape sequences. For instance, `\ ` is not valid, which becomes an identity escape sequence in Unicode-unaware mode, but an error in Unicode-aware mode.

`invalidCharacters` is *only* used to form this regex, so if you want to avoid lodash I would just construct this regex literally and use .source to get the value (to avoid escaping issues).

```suggestion
export const oneValidCharacterRe = /[^/*?[\]!\\~|#"' ]/;
```